### ISSUE TITLE
ConditionalFilter.js: text input doesn't have an id or aria-label specified

### ISSUE THREAD

**Issue by gkarat** [RedHatInsights/frontend-components#960]
It seems that `<Text />` component under `<div className="ins-c-conditional-filter">` (see code reference below) doesn't receive neither `id` or `area-label`. It causes an error in the console:
`Text input: Text input requires either an id or aria-label to be specified`

https://github.com/RedHatInsights/frontend-components/blob/997a844bb2d5450bb0edce2750b28942cbecdb6e/packages/components/src/ConditionalFilter/ConditionalFilter.js#L43

**Comment by gkarat** [comment]
Here is evidence from one of the applications using this component: 
![Screenshot from 2021-02-11 13-23-55](https://user-images.githubusercontent.com/31385370/107636165-85ae6a00-6c6c-11eb-97c4-218860670951.png)

**Comment by rvsia** [comment]
Hello, I think you can set it manually as a prop.

This is an example from Sources UI https://github.com/RedHatInsights/sources-ui/blob/9d4c77916058ed0d3cc6ba18f2008227102fb78c/src/pages/Sources.js#L188 

```jsx
            {
              label: intl.formatMessage({
                id: 'sources.name',
                defaultMessage: 'Name',
              }),
              filterValues: {
                'aria-label': intl.formatMessage({
                  id: 'sources.filterByName',
                  defaultMessage: 'Filter by name',
                }),
                onChange: (_event, value) => {
                  stateDispatch({ type: 'setFilterValue', value });
                  debouncedFiltering(() => setFilter('name', value, dispatch));
                },
                value: filter,
              },
            },
```

**Comment by gkarat** [comment]
Thank you, that should solve the problem :-)
